ncm-authconfig/src/main/pan/components/authconfig/schema.pan

```diff
@@ -17,7 +17,7 @@ type yesnostring = string with match(SELF, "yes|no");
 
 type authconfig_pamadditions_line_type = {
   "order"       : string with match(SELF, '^(first|last)$')
-  "entry"       : string with match(SELF, '\S+\.so$')
+  "entry"       : string with match(SELF, '^\s*\w+\s+\S+\.so(\s|$)')
```

Comment:
the regex you propose is not that much simpler, and allows typos like `required      pam_access.soaccessfile=/etc/security/access_sshd.conf`